engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -33,13 +33,8 @@ static BOOL DoesHardwareSupportWideGamut() {
   static dispatch_once_t once_token = 0;
   dispatch_once(&once_token, ^{
     id<MTLDevice> device = MTLCreateSystemDefaultDevice();
-    if (@available(iOS 13.0, *)) {
-      // MTLGPUFamilyApple2 = A9/A10
-      result = [device supportsFamily:MTLGPUFamilyApple2];
-    } else {
-      // A9/A10 on iOS 10+
-      result = [device supportsFeatureSet:MTLFeatureSet_iOS_GPUFamily3_v2];
-    }
+    // MTLGPUFamilyApple2 = A9/A10
```

Comment:
Optional: Remove this comment while you are here, as it appears to be [actively wrong](https://developer.apple.com/documentation/metal/mtlgpufamily/apple2)? (Docs say it's A8+, so not only is the fact that this sounds like it's reporting a max misleading, but the min isn't correct.)

Reply:
Done. Good catch!